benches/capture/src/main.rs

```diff
@@ -48,6 +48,7 @@ fn capture(source_root: &Path, dest: &Path, force: bool) {
         .write(dst, Compression::best());
     let mut ar = tar::Builder::new(encoder);
     ar.mode(tar::HeaderMode::Deterministic);
+    ar.sparse(false);
```

Comment:
This is for internal benchmark only, so should be fine with sparse as well.

Reply:
Not sure what this bench exactly does, but my thought was that it should be closer to the real-world scenario as checking each file to sparseness might (or might not) affect performance.

Reverted.

Reply:
These tarballs are snapshots of some real world Cargo workspaces, we don't benchmark on tar/untar of them.

https://github.com/rust-lang/cargo/blob/b5d44db1daf0469b227a6211b987162a39a54730/benches/capture/src/main.rs#L1-L5

Anyhow, thanks for the revert!